components/brave_new_tab_ui/hooks/PrefHookManager.ts

```diff
@@ -27,7 +27,7 @@ export class PrefHookManager<T> {
         getInitialValue().then(this.notifyListeners)
     }
 
-    notifyListeners (prefs: T) {
+    notifyListeners = (prefs: T) => {
```

Comment:
Just wondering why this change is neccessary?

Reply:
I think there was a case where the this context was being set to something else